static/app/components/events/interfaces/frame/codecovLegend.tsx

```diff
@@ -0,0 +1,90 @@
+import {useMemo} from 'react';
+import styled from '@emotion/styled';
+
+import {IconCircle} from 'sentry/icons';
+import {IconCircleFill} from 'sentry/icons/iconCircleFill';
+import {t} from 'sentry/locale';
+import space from 'sentry/styles/space';
+import {Event, Frame, Organization} from 'sentry/types';
+import {CodecovStatusCode} from 'sentry/types/integrations';
+import useProjects from 'sentry/utils/useProjects';
+
+import useStacktraceLink from './useStacktraceLink';
+
+interface CodecovLegendProps {
+  event: Event;
+  frame: Frame;
+  nextFrame: Frame | undefined;
+  organization: Organization | undefined;
+}
+
+export function CodecovLegend({
+  event,
+  frame,
+  organization,
+  nextFrame,
+}: CodecovLegendProps) {
+  const {projects} = useProjects();
+  const project = useMemo(
+    () => projects.find(p => p.id === event.projectID),
+    [projects, event]
+  );
+
+  const {data: match, isLoading} = useStacktraceLink({
```

Comment:
calling data match here probably doesn't make as much sense. not sure why we did that in stacktrace link